packages/devtools_app/lib/src/vm_developer/vm_developer_common_widgets.dart

```diff
@@ -0,0 +1,121 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import '../common_widgets.dart';
+import '../table.dart';
+import '../theme.dart';
+
+/// A convenience widget used to create non-scrollable information cards.
+///
+/// `title` is displayed as the header of the card and is required.
+///
+/// `rowKeyValues` takes a list of key-value pairs that are to be displayed as
+/// individual rows. These rows will have an alternating background color.
+///
+/// `table` is a widget (typically a table) that is to be displayed after the
+/// rows specified for `rowKeyValues`.
+class VMInfoCard extends StatelessWidget {
+  const VMInfoCard({
+    @required this.title,
+    this.rowKeyValues,
+    this.table,
+  });
+
+  final String title;
+  final List<MapEntry> rowKeyValues;
+  final Widget table;
+
+  @override
+  Widget build(BuildContext context) {
+    return Flexible(
```

Comment:
Can you remove the flexible and declare it upstream? It is more intuitive for `Flexible` widgets to be declared as part of their parent Flex widget (row, column, etc.), that way the flex value can be modified and the flex layout of the parent is more clear from the build method

Reply:
Sure, will do.